about.html

```diff
@@ -16,22 +16,55 @@
 
   <header>
     <navbar class="nav-dsk">
-      <a href="index.html"><img src="images/logo.png" alt="" class="logo"></a>
+      <a href="index.html"><img src="images/logo.png" alt="logo" class="logo"></a>
+      <div class="dark-nav">
+        <ul>
+          <li><a href="#"><i class="fa-brands fa-facebook-f"></i></a></li>
+          <li><a href="#"><i class="fa-brands fa-twitter"></i></a></li>
+          <li><a href="#">English</a></li>
+          <li><a href="#">My Page</a></li>
+          <li><a href="#">Logout</a></li>
+        </ul>
+      </div>
       <ul>
-        <li><a href="index.html">Home</a></li>
         <li><a href="about.html">About</a></li>
+        <li><a href="about.html">Program</a></li>
+        <li><a href="about.html">Join</a></li>
+        <li><a href="about.html">Sponsors</a></li>
+        <li><a href="about.html">News</a></li>
         <li class="menu-highlight"><a href="#">Stoa Campaign</a></li>
       </ul>
     </navbar>
     <!-- Menu Mobile-->
-    <navbar class="nav"><img src="images/mobile_menu.png" alt="hamburger"></navbar>
+    <nav>
+      <a class="hamburger">
+        <img src="images/mobile_menu.png" alt="hamburger">
+      </a>
+    </nav>
+    <nav>
+
+      <ul class="mobile-menu">
+        <a class="close"><img src="images/cancel.png" alt=""></a>
+        <li class="menu-link"><a href="index.html">Home</a></li>
+        <li><a href="about.html">About</a></li>
```

Comment:
- [optional] kindly make the main section of the about page full width. 
![Screenshot (886)_LI](https://user-images.githubusercontent.com/65298048/226122627-5fcb0538-0109-47ca-ac68-8e7dce5f499d.jpg)